project/app/api/getcounthelper.py

```diff
@@ -0,0 +1,93 @@
+def helper(df):
+    count = {}
+    for set_of_tags in df["tags"]:
+        for tag in set_of_tags:
+            if tag in count:
+                count[tag] += 1
+            else:
+                count[tag] = 1
+
+    force = {}

```

Comment:
For the future, let's see some more commenting. This looks like 'our scale' re police brutatlity.